docs/guides/auth.md

```diff
@@ -38,6 +40,8 @@ Save the client ID and secret as the environment variables HCP_CLIENT_ID and HCP
 
 Or, configure the provider with the client ID and secret by copy-pasting the values directly into provider config.
 
+-> **Warning:** Hard-coded credentials are not recommended in Terraform configuration outside of local testing and can risks secret exposure if committed to a code repository.
```

Comment:
👀 Small thing I noticed: I think you may have intended to put `can risk` or `risks` here 😄